src/components/chip/chip.tsx

```diff
@@ -251,13 +259,19 @@ export class Chip implements ChipInterface {
         return <span class="text">{this.text}</span>;
     };
 
-    private renderIcon() {
+    private renderPicture() {
         const icon = getIconName(this.icon);
 
-        if (!icon) {
+        if (!icon && !this.image) {
             return;
         }
 
+        if (this.image) {
+            return (
+                <img src={this.image.src} alt={this.image.alt} loading="lazy" />
```

Comment:
are we or should we be concerned about svgs with embedded scripts, and generally all the bad things people can do with images?

Reply:
If loading an external image in an `img` element can be an attack vector, I guess we need to consider it at least. I didn't get so far as to investigate that. I would hope that embedded scripts in svgs wouldn't be executed, or at least not executed in a way that gains any access to the rest of the page…? 😬